### PR TITLE
Filter Services with no perfdata

### PR DESCRIPTION
**Pull request by Virsacer**
Hi,
services that don't have perfdata can be removed from the list.

This is my first pull-request ever, so I hope I did it correctly...

### CODE REVIEW

File: application/controllers/ListController.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 use Icinga\Module\Monitoring\DataView\DataView;
 use Icinga\Module\Monitoring\Object\Host;
 use Icinga\Module\Monitoring\Object\Service;
+use Icinga\Data\Filter\Filter;
```
Comment (Al2Klimov) [inline application/controllers/ListController.php:13]: Please keep the uses sorted alphabetically.

File: application/controllers/ListController.php [hunk 2]
```diff
@@ -76,6 +77,8 @@ public function servicesAction()
             $this->backend->select()->from('servicestatus', ['host_name', 'service_description'])
         );
 
+        $servicesQuery->applyFilter(Filter::expression('service_perfdata', '!=', ''));
+
```
Comment (Al2Klimov) [inline application/controllers/ListController.php:81]: Please do the same for hosts.

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Hello @Virsacer and thank you for contributing!

Please provide before/after screenshots once you've addressed my change requests.

Also don't forget to use `git commit --amend`.

Best,
AK

**Comment by Virsacer** [comment]
Before:
image

After:
image

**Review by Al2Klimov — CHANGES_REQUESTED** [review]
Please also provide hosts screenshots.

**Comment by Virsacer** [comment]
@Al2Klimov My host-checks all have performance-data so there would be no visible difference.
And if they hadn't then the difference would be the same as in the services screenshot - just with diffenrent names...

**Comment by Al2Klimov** [comment]
Does the hosts view at least work at all with the currently pushed commit?

**Comment by Virsacer** [comment]
Ofcourse it does :-)

**Review by Al2Klimov — APPROVED** [review]
Fine.

**Review by bobapple — APPROVED** [review]

**Pull request merged**